src/shared/ui-kit/components/TermsConditionModal/TermsConditionModal.tsx

```diff
@@ -0,0 +1,178 @@
+import { Button, Flex, Modal, ModalBody, ModalContent, ModalHeader, ModalOverlay } from "@chakra-ui/react";
+import { Color, Text } from "@shared/ui-kit";
+import { LocalStorageKeys } from "../constants";
+import { useState } from "react";
+
+export function TermsConditionModal() {
+  const localStorageState = localStorage.getItem(LocalStorageKeys.ACCEPTED_TERMS_AND_CONDITIONS);
+  const [acceptedCondition, setAcceptedCondition] = useState(localStorageState ? JSON.parse(localStorageState) : false);
+
+  const onAccept = () => {
+    setAcceptedCondition(true);
+    localStorage.setItem(LocalStorageKeys.ACCEPTED_TERMS_AND_CONDITIONS, JSON.stringify(true));
+  };
+
+  if (acceptedCondition) {
+    return <></>;
+  }
+  return (
+    <Modal
+      variant="termsConditionModal"
+      isCentered
+      isOpen={true}
+      onClose={() => {
+        console.log("[ANS] close modal");
+      }}
+    >
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>
+          <Flex direction="row" alignItems="center" justifyContent="center">
+            <Text.P_Large_Semibold>Terms And Conditions</Text.P_Large_Semibold>
+          </Flex>
+        </ModalHeader>
+        <ModalBody>
+          <Flex direction="column" alignItems="flex-start" justifyContent="center" gap={6}>
+            <Text.P_Medium_Medium style={{ maxHeight: "7.5rem", overflowY: "auto" }}>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam eu metus in eros lobortis porta eget sed
+              sapien. Maecenas tristique ultrices sapien id aliquam. Pellentesque sollicitudin bibendum risus. Lorem
+              ipsum dolor sit amet, consectetur adipiscing elit. Aliquam eu metus in eros lobortis porta eget sed
+              sapien. Maecenas tristique ultrices sapien id aliquam. Pellentesque sollicitudin bibendum risus. Lorem
+              ipsum dolor sit amet, consectetur adipiscing elit. Aliquam eu metus in eros lobortis porta eget sed
+              sapien. Maecenas tristique ultrices sapien id aliquam. Pellentesque sollicitudin bibendum risus.
+            </Text.P_Medium_Medium>
+            <Button key="open-mint-nft" variant="primary" width="100%" onClick={onAccept}>
```

Comment:
button `key` should be `relatable and unique`

Reply:
Done.